updates/13.sql

```diff
@@ -0,0 +1,4 @@
+SET SESSION sql_mode='NO_AUTO_VALUE_ON_ZERO';
```

Comment:
can we remove the NO_AUTO_VALUE_ON_ZERO change?

Reply:
I think we need it in this case, because the id column contains '0' values. 
Otherwise an error is thrown.

https://dev.mysql.com/doc/refman/8.0/en/sql-mode.html#sqlmode_no_auto_value_on_zero

Reply:
As far as i understand the SQL mode, it can consist of several settings at the same time. The default depends on the server settings.

E.g. on my machine this is a special value which is documented in the project's `README.md`: `STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION`

If you use the `SET SESSION` command, you will set the value to  `NO_AUTO_VALUE_ON_ZERO`, but therefore lose all other existing changes like e.g. `STRICT_TRANS_TABLES` which might as well be necessary for the migration.
The other aspect is that we might execute multiple updates during the same session. So you should do the following

1) Find a way to add this setting to the existing ones, instead of entirely overriding it
2) Save the previous value in a variable and restore it after the migration

Reply:
I have found a way to restore the sql_mode

```
-- restore sql mode
SET SESSION sql_mode=default;
```

Reply:
some more changes please:
- Changing the FOREIGN KEY doesnt require the sql_mode to be changed, so please move the first SET SESSION sql_mode statement directly above the introduction of AUTO_INCREMENT
- AUTO_INCREMENT is a sql keyword and should be written in uppercase
- 13.sql already existed and contained a change which had nothing to do with the AUTO_INCREMENT change. Moreover there have been additional .sql files created in the meantime and already been applied on certain machines. Please move the changes related to selektion_quellengattung and the FOREIGN KEY to a new separate file (i think 17.sql should the next non-existing number in the sequence.)

Reply:
1) moved to set auto_increment
2)fixed
3)it is now in 17.sql

Reply:
This aspect is still not solved:

> Find a way to add this setting (NO_AUTO_VALUE_ON_ZERO) to the existing ones, instead of entirely overriding it

Reply:
Now the CONCAT Sql function will be used to append the sql_mode to the existing ones